packages/typespec-go/test/armdatabasewatcher/zz_models.go

```diff
@@ -78,42 +78,6 @@ type Datastore struct {
 	KustoClusterDisplayName *string
 }
 
-// ErrorAdditionalInfo - The resource management error additional info.
-type ErrorAdditionalInfo struct {
```

Comment:
This, and other types, were inadvertently resurrected in the last big toolset upgrade due to breaking changes in tcgc. This has now been rectified.